ansible/group_vars/omero-hosts.yml

```diff
@@ -514,20 +514,24 @@ omero_web_apps_config_set:
           href: "https://idr.openmicroscopy.org/about/index.html"
         - text: "Published studies"
           href: "https://idr.openmicroscopy.org/about/studies.html"
-        - text: "Linked resources"
-          href: "https://idr.openmicroscopy.org/about/linked-resources.html"
         - text: "API Access"
           href: "https://idr.openmicroscopy.org/about/api.html"
         - text: "Data download"
           href: "https://idr.openmicroscopy.org/about/download.html"
-        - text: "Image Tools Resource (ITR)"
-          href: "https://idr.openmicroscopy.org/about/itr.html"
-        - text: "Analysis Environments"
-          href: "https://idr.openmicroscopy.org/about/analysis-environments.html"
         - text: "Deployment"
           href: "https://idr.openmicroscopy.org/about/deployment.html"
         - text: "FAQ"
           href: "https://idr.openmicroscopy.org/about/faq/"
+    - text: "Resources"
+      submenu:
+        - text: "IDR images as OME-NGFF"
```

Comment:
Given the target link,  I think either `OME-NGFF samples` or `OME-NGFF examples` might be more suitable. I am not convinced we need to repeat `IDR` as it's implicit for every menu element.

Two other thoughts:
- the converted datasets might be collections of images either multi-image or plates
- this link points to representative NGFF data derived from IDR data rather than "All IDR data converted to OME-NGFF" which the current title might be suggesting

Reply:
Entries updated on idr-testing to ```OME-NGFF examples```